### ISSUE TITLE
2025-10-29: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#373]
# 每日更新报告（2025-10-29）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-10-29 23:10:35 | Nicolò Lucchesi | [CI] Fix flaky `test_two_responses_with_same_prev_id` test  (#27745) |
| 2025-10-29 22:44:03 | Xiake Sun | [ROCm][Platform] Add MI308X device id in _ROCM_DEVICE_ID_NAME_MAP (#27623) |
| 2025-10-29 21:01:05 | Roger Young | Fix MiniMax-M2 rmsnorm precision and remove useless code (#27627) |
| 2025-10-29 20:59:48 | Cyrus Leung | [Frontend] Add `vllm bench sweep` to CLI (#27639) |
| 2025-10-29 20:55:51 | Zhewen Li | [CI/Build] Skip cpu offloading test on AMD (#27690) |
| 2025-10-29 20:19:37 | Isotr0py | [VLM] Add Qwen3-VL generation test (#25185) |
| 2025-10-29 18:29:49 | Kevin H. Luu | [chore] Remove models weight on S3 logic (#27725) |
| 2025-10-29 18:24:52 | Eugene Khvedchenya | Feature/video support in random mm dataset (#25963) |
| 2025-10-29 18:06:42 | Isotr0py | [Misc] Raise error for missing video metadata in `MultiModalDataParser` (#27664) |
| 2025-10-29 18:03:54 | Yue Zhang | [Fix] import get_kv_cache_torch_dtype error in LMCacheConnector integration (#27670) |
| 2025-10-29 18:01:32 | Alec S | [Frontend] [gpt-oss] Mcp type bug (#27689) |
| 2025-10-29 17:42:44 | Alec S | [Frontend] [gpt-oss] Tool json call parsing error retry (#27675) |
| 2025-10-29 16:14:33 | bnellnm | [Bugfix] Fix modular kernel tests (#27707) |
| 2025-10-29 16:12:54 | Jiangyun Zhu | [perf] Enable concurrent execution of "shared_experts" and "selected_experts" in qwen3-next (#27578) |
| 2025-10-29 16:04:33 | Cyrus Leung | [CI/Build] Move pre-commit only scripts to `tools/pre_commit` (#27657) |
| 2025-10-29 15:54:21 | Dipika Sikka | [Speculators] Move tests + fix integration (#27308) |
| 2025-10-29 15:00:15 | Zhewen Li | [Bugfix] Fix non-contiguous tensor error in `rocm_unquantized_gemm_impl` (#27605) |
| 2025-10-29 14:27:30 | Zhewen Li | [CI/Build][Bugfix]Fix Quantized Models Test on AMD (#27712) |
| 2025-10-29 13:38:37 | Shaoting | [KV Connector] Update lmcache connector with latest compatibility (#27681) |
| 2025-10-29 13:28:20 | Lukas Geiger | [Model] Fix Qwen3VL and Qwen3Omni after torch.compile changes (#27705) |
| 2025-10-29 12:43:29 | liuzhenwei | [NIXL][XPU] update name of nixl wheel (#27631) |
| 2025-10-29 12:07:09 | Varun Sundar Rabindranath | [Build] Revert triton_kernels requirements (#27659) |
| 2025-10-29 11:58:01 | Simon Mo | Revert "Install pre-built xformers-0.0.32.post2 built with pt-2.9.0" (#27714) |
| 2025-10-29 10:39:15 | Huy Do | Install pre-built xformers-0.0.32.post2 built with pt-2.9.0 (#27598) |
| 2025-10-29 09:32:33 | Jialin Ouyang | [Core] Early return in SlidingWindowManager.remove_skipped_blocks (#27673) |
| 2025-10-29 07:53:12 | Wentao Ye | [Bug] Fix deepep low latency use nvlink by default (#27677) |
| 2025-10-29 06:36:43 | Lucas Kabela | [Misc][qwen2_5_vl][torch.compile] Enable `supports_torch_compile` on generic nn.Module and demonstrate speedup on Qwen Vision model (#23207) |
| 2025-10-29 06:35:54 | Nick Hill | [AsyncScheduling] Make async overlap work with logprobs (#27615) |
| 2025-10-29 05:01:33 | Or Ozeri | [Core] Scheduler: Publish connector events after output (#25875) |
| 2025-10-29 04:51:35 | Wentao Ye | [Test] Batch Invariant: Unit test using parameterized backend (#27478) |
| 2025-10-29 01:55:10 | Lucas Wilkinson | [Misc] Make reorder batch also separate extends (#27367) |
| 2025-10-29 01:49:15 | Matvei Pashkovskii | Add load pattern configuration guide to benchmarks (#26886) |
| 2025-10-29 00:54:26 | Zhengxu Chen | [compile] Add fallback path to AOT compile when serialization fails. (#27350) |
| 2025-10-29 00:23:35 | Cyrus Leung | [Misc] Make `LayerBlockType` a `Literal` instead of `Enum` (#27658) |
